index.html

```diff
@@ -237,18 +237,21 @@ <h2>
           To <dfn>block a permission</dfn>, run these steps:
         </p>
         <ol class="algorithm">
+          <li>Let |document:Document| be the <a>responsible document</a> of the
+          <a>current settings object</a>.
+          </li>
           <li>Let |descriptor:WakeLockPermissionDescriptor| be an instance of
-            {{WakeLockPermissionDescriptor}}.
+          {{WakeLockPermissionDescriptor}}.
           </li>
           <li>Set |descriptor|'s <a>permission state</a> to {{"denied"}}.
           </li>
           <li>Let |record| be the <a>platform wake lock</a>'s <a>state record</a>
-          associated with |descriptor|'s <a data-lt=
-          "WakeLockPermissionDescriptor.type">type</a> member.
+          associated with |document| and |descriptor|'s <a data-lt=
+          "WakeLockPermissionDescriptor.type">type</a> member |type|.
```

Comment:
Aren't you duplicating `type` here?

Reply:
Maybe, but I needed type variable so that I can refer to it later - maybe it should be written in another way